templates/general/index.html

```diff
@@ -0,0 +1,249 @@
+{% extends 'base.html' %}
+
+{% block head %}
+{{ super() }}
+<script src="js/index/tab_headers.js"></script>
```

Comment:
Import should be relative to domain root.
Add "/" in front of the string ("/js/...")